backtracking/graph_coloring.cpp

```diff
@@ -17,29 +17,38 @@
  * @author [Anup Kumar Panwar](https://github.com/AnupKumarPanwar)
  * @author [David Leal](https://github.com/Panquesito7)
  */
-#include <array>
-#include <iostream>
-#include <vector>
+
+#include <array>     /// for std::array
+#include <iostream>  /// for IO operations
+#include <vector>    /// for std::vector
 
 /**
- * @namespace
+ * @namespace backtracking
  * @brief Backtracking algorithms
  */
 namespace backtracking {
-/** A utility function to print solution
+/**
+ * @namespace graph_coloring
+ * @brief Functions for the [Graph
+ * Coloring](https://en.wikipedia.org/wiki/Graph_coloring) algorithm,
+ */
+namespace graph_coloring {
+/**
+ * @brief A utility function to print the solution
  * @tparam V number of vertices in the graph
  * @param color array of colors assigned to the nodes
  */
 template <size_t V>
 void printSolution(const std::array<int, V>& color) {
-    std::cout << "Following are the assigned colors" << std::endl;
+    std::cout << "Following are the assigned colors\n";
     for (auto& col : color) {
         std::cout << col;
     }
-    std::cout << std::endl;
+    std::cout << "\n";
```

Comment:
Why?

Reply:
According to what @ayaankhan98 has said previously, `\n` is faster than `std::endl`.